pages/record/_.vue

```diff
@@ -9,31 +9,53 @@
     data-qa="record page"
     class="mt-5"
   >
-    <b-row class="mb-3 mx-0 card p-3">
+    <b-row class="mb-3">
       <b-col
         cols="12"
-        class="mb-3 px-0"
+        lg="9"
       >
-        <MediaPresentation
-          :codec-name="edmIsShownBy.edmCodecName"
-          :image-link="image.link"
-          :image-src="image.src"
-          :mime-type="edmIsShownBy.ebucoreHasMimeType"
-          :url="edmIsShownBy.about"
-          :width="edmIsShownBy.ebucoreWidth"
-          :height="edmIsShownBy.ebucoreHeight"
-        />
+        <div class="card p-3">
+          <div
+            class="card-grid"
+            :class="isRichMedia && 'card-grid-richmedia'"
+          >
+            <h1
+              v-if="title"
+            >
+              {{ title | inCurrentLanguage($i18n.locale) }}
+            </h1>
+            <MediaPresentation
+              :codec-name="edmIsShownBy.edmCodecName"
+              :image-link="image.link"
+              :image-src="image.src"
+              :mime-type="edmIsShownBy.ebucoreHasMimeType"
+              :url="edmIsShownBy.about"
+              :width="edmIsShownBy.ebucoreWidth"
+              :height="edmIsShownBy.ebucoreHeight"
+              class="mb-3"
+            />
+            <p
+              v-if="description"
+              class="description"
+            >
+              {{ description | inCurrentLanguage($i18n.locale) }}
```

Comment:
For description (only) we want to honour white space to permit presentation of paragraphs, e.g. in the record /125/contributions_42580010_c414_0137_3824_6eee0af7162b

I would suggest trying the CSS `white-space: pre-wrap;` which has the additional benefit of presenting poetry as intended.

Reply:
`white-space: pre-wrap;` adds some additional (extra whitespace before the first line of text and indentation - always - of the first line) styling, which I couldn't get around. I tried to match the behaviour on the current portal and updated the filter accordingly.